### DISCUSSION TITLE
Does it support  the tensorflow version

### DISCUSSION THREAD

**Discussion by atztao** [zju3dv/LoFTR#24] [Q&A]

**Comment by JiamingSuen** [comment]
Hi @atztao, unfortunately no. We don't have plans for a TensorFlow port in near future.

**Comment by RohanG9929** [comment]
Hi there!

My team and I were able to create a Tensorflow implementation. It still needs some work but we believe this is a great foundation to build upon.

https://github.com/RohanG9929/LoFTR-in-Tensorflow
